knowledge-base/how-to-set-the-map-extent-dynamically-when-showing-a-single-data-point.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Set Map extent dynamically when showing a single data point
+description: Explains how to set map extent dynamically when showing a single data point
+type: how-to
+page_title: How to Set the Map Extent Dynamically When Showing a Single Data Point
+slug: how-to-set-the-map-extent-dynamically-when-showing-a-single-data-point
+position: 
+tags: map
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The article explains how to set the map extent dynamically when showing a single data point.
```

Comment:
Provide more detailed description. Why is this useful, in what scenarios it can help and how? What makes it difficult to achieve and why it is not an out of the box thing to achieve? Avoid repeating the title in this section as this shows hastiness and doesn't speak well of the KB.